documentation/adr/0002-wiki-platform.md

```diff
@@ -0,0 +1,294 @@
+
+#### Details
+
+- **Hosting:** SaaS
+- **Pricing:** [$5.75 (standard) or $11 (premium) per user per month](confluence-pricing)
+- **Content Review:** Supported through drafts, not enforceable
+- **Supported Media:** TODO
+- **I18n:** Limited third party plugins for automating translation
+- **Web Analytics:** [Google Analytics plugin](confluence-ga) available, also native analytics with premium tier
+- **Open Source Status:** Propietary
+- **External Contributions:** Only supported in public spaces (without review)
+
+#### Pros
+
+- Relatively user friendly for non-technical users
+- Supports a wide variety of media and page content
+- Supports drafts, which allow edits to be made without publishing
+- Supports page history and comparison of previous versions
+- Limited support for public pages and spaces
+- Minimimal ongoing maintenance costs due to SaaS hosting
+- Most affordable per user cost for a given tier of features
+
+#### Cons
+
+- Does not support a formal review process for drafts before they can be published
+- Supports fewer content types than Notion, especially in terms of structured data
+- Content can only be made public at the level of an individual page or an entire space
+- Content API is limited in capability and less intuitive than Notion API
+- No direct support for internationalization and localization
+- Closed source proprietary tool
+- Data is controlled by Confluence, only accessible via API
+
+### Notion
+
+[Notion](notion) is a Software as a Service (SaaS) documentation and collaboration tool that also allows users to add structured and semi-structured content to pages. Because Notion offers a fully-featured API for reading and managing content, it also has a robust set of community integrations that extend Notion's core functionality.
+
+#### Details
+
+- **Hosting:** SaaS
+- **Pricing:** [$8 (pro) or $15(business) per user per month](notion-pricing)
+- **Content Review:** Not supported
+- **Supported Media:** TODO
+- **I18n:** Third-party beta plugin for translation
+- **Web Analytics:** TODO
+- **Open Source Status:** Propietary
+- **External Contributions:** Supported in app (without review)
+
+#### Pros
+
+- Relatively user friendly for non-technical users
+- Allows publishing pages to the web for external user access
+- Supports the widest variety of page content and media
+- Supports public comments and edits in the app
+- Minimimal ongoing maintenance costs due to SaaS hosting
+- Exposes wiki content via an API
+- Robust plugin and add-on community
+
+#### Cons
+
+- Slightly more complicated interface than Confluence
+- Pages published to the web aren't organized as clearly as public pages in GitBook
+- No content review process, all changes and comments are published automatically
+- No direct support for internationalization and localization
+- Page history is limited to 30 (pro) or 90 (business) days
```

Comment:
oof

Reply:
Yeah I was impressed by a lot of Notion features, especially their API, but the cons are pretty bad 😞